bootstrap.py

```diff
@@ -39,8 +42,8 @@
                   "is to show the console")
 parser.add_option('--hide-console', action='store_true',
                   default=False, help="Hide parent console window (Windows only)")
-parser.add_option('--test', dest="test", action='store_true', default=False,
-                  help="Test Spyder with a clean settings dir")
+parser.add_option('--clean', dest="clean", action='store_true', default=False,
```

Comment:
Please change this to `--clean-config`. The problem is `--clean` is too close in meaning to `--reset`.

Reply:
Good point @ccordoba12 . On that note, ``--clean-config`` could also be misinterpreted in the same way, i.e. that the option would ``clean`` the ``config`` i.e. what ``reset`` does. What about ``--clean-mode``? That would make it much clearer that its temporary and distinct from ``reset``.

Reply:
Firefox call it `--safe-mode`. What about that?

Reply:
You're right about `--clean-config`, by the way.

Reply:
Done.